File: test/export/js/07-subprocesos.js
function sin_titulo() {
	var c;
	document.write(funciondoble(5),'<BR/>');
	noretornanada(3,9);
	c = 0;
	porreferencia(c);
	document.write(c,'<BR/>');
	var a = new Array(10);
	var b = new Array(3); for (var aux_index_0=0;aux_index_0<3;aux_index_0++) { b[aux_index_0] = new Array(4); }
	recibevector(a);
	recibematriz(b);
}
function funciondoble(b) {
	var a;
	a = 2*b;
	return a;
}
function noretornanada(a, b) {
	document.write(a+b,'<BR/>');
}
function porreferencia(b) {
	b = 7;
}
function recibevector(v) {
	v[0] = prompt();
}
function recibematriz(m) {
	m[0][0] = prompt();
}

File: test/export/java/07-subprocesos.java
import java.io.*;
public class sin_titulo {
	BufferedReader bufEntrada = new BufferedReader(new InputStreamReader(System.in));
	public static void main(String args[]) throws IOException {
		String a[];
		String b[][];
		double c;
		System.out.println(funciondoble(5));
		noretornanada(3,9);
		c = 0;
		porreferencia(c);
		System.out.println(c);
		a = new String[10];
		b = new String[3][4];
		recibevector(a);
		recibematriz(b);
	}
	public static double funciondoble(double b) {
		double a;
		a = 2*b;
		return a;
	}
	public static void noretornanada(String a, String b) {
		System.out.println(a+b);
	}
	public static void porreferencia(double b) {
		b = 7;
	}
	public static void recibevector(String v[]) throws IOException {
		v[0] = bufEntrada.readLine();
	}
	public static void recibematriz(String m[][]) throws IOException {
		m[0][0] = bufEntrada.readLine();
	}
}
